app/services/CollectionService.scala

```diff
@@ -63,34 +63,28 @@ object CollectionService {
     }
   }
 
-  def getStoriesForCollectionStages(collectionId: String, collection: CollectionJson, config: ConfigJson): (Seq[Story], Seq[Story]) = {
-    val numberOfGroups = config.collections.get(collectionId) match {
-      case Some(collectionConfig) => collectionConfig.groups.map(_.length).getOrElse(0)
-      case None => 0
-    }
-
-    val liveStages = Some(collection.live).map { getStoriesForStage(_, numberOfGroups) }.getOrElse(Seq())
+  def getStoriesForCollectionStages(collectionId: String, collection: CollectionJson): (Seq[Story], Seq[Story]) = {
+    val liveStages = Some(collection.live).map(getStoriesForStage).getOrElse(Seq())
 
     (
       liveStages,
```

Comment:
we do we don't get the liveStages inline like we do draftStages?

Reply:
Yeah, maybe it's a bit gross -- I've refactored.